controllers/UserController.js

```diff
@@ -0,0 +1,44 @@
+const apiResponse = require("../helpers/apiResponse");
+const UserModel = require("../models/UserModel");
+const { validationResult } = require("express-validator");
+const auth = require("../middlewares/jwt");
+
+exports.getUser = [
+  auth,
+  (req, res) => {
+    try {
+      const user = req.user;
+
+      const errors = validationResult(req);
+      if (!errors.isEmpty()) {
+        return apiResponse.validationErrorWithData(
+          res,
+          "Validation Error.",
+          errors.array()
+        );
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 tabs but found 8 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
				);
```